docs/source/llms/llm-evaluate/index.rst

```diff
@@ -358,7 +358,7 @@ the score is "yes" otherwise "no".
 
 .. code-block:: python
 
-    def eval_fn(predictions, targets, metrics):
+    def eval_fn(predictions, targets):
```

Comment:
I feel like it's less confusing if we remove "metrics" and recommend that users put the name of the metric as the argument/parameter, so there is only one way to access dependent metrics. Thoughts? (Previously only having predictions, targets as arguments didn't work which is why we said you must have metrics, but this is no longer true).

Reply:
I only updated this here but can update this across all our documentation if we decide that's what we want to recommend.

Reply:
I agree - can we update this across our docs?